docs/api.md

```diff
@@ -350,6 +350,66 @@ class ResetPasswordPage extends React.Component {
 }
 ```
 
+#### ChangePasswordForm
+
+Renders a change password form. The parameter `spToken` is required in order for the token to be validated.
+
+```html
+<ChangePasswordForm spToken={this.props.location.query.sptoken} />
+```
+
+Customize the form by providing your own markup.
+
+```html
+<ChangePasswordForm>
```

Comment:
If the `spToken` parameter was optional I would agree with you, but since `<ChangePasswordForm>` is kind of invalid without it, I think it's bad to have that in an example. It happened to me a couple of times that I just copied this custom form and forgot to add the parameter afterwards.

#### Default form:

```jsx
<ChangePasswordForm spToken={this.props.location.query.sptoken} />
```

#### Custom form:

```jsx
<ChangePasswordForm spToken={this.props.location.query.sptoken}>
  ...
</ChangePasswordForm>
```